src/plugins/foxPage/utils.ts

```diff
@@ -122,3 +122,13 @@ export const rewardRatePerToken = memoize(async (farmingRewardsContract: Contrac
     throw new Error(errorMsg)
   }
 })
+
+export const makeLpApr = (foxRewardRatePerToken: string, foxEquivalentPerLPToken: string) =>
```

Comment:
maybe makeTotalLpApr is a better some as it is the combination of farming rate + LP rate ?

Reply:
Good idea! Always better to go explicit vs. broad naming, updated in  4afa9188a9f1dbc0fc9218ed6208e5e2965b6a4b